en/deploy-tidb-from-kubernetes-gke.md

```diff
@@ -201,15 +193,15 @@ kubectl -n demo edit tc basic
 
 ## Access the Grafana dashboard
 
-To access the Grafana dashboards, you can forward a port from the Cloud Shell to the Grafana service in Kubernetes. (Cloud Shell already uses port 3000 so we use port 3003 in this example instead.)
+To access the Grafana dashboards, you can forward a port from the Cloud Shell to the Grafana service in Kubernetes. (Cloud Shell already uses port 3000 so we use port 8080 in this example instead.)
 
 To do so, use the following command:
 
 ```shell
-kubectl -n demo port-forward svc/basic-grafana 3003:3000 &>/tmp/pf3003.log &
+kubectl -n demo port-forward svc/basic-grafana 8080:3000 &>/tmp/pf8080.log &
 ```
 
-Open this URL to view the Grafana dashboard: <https://ssh.cloud.google.com/devshell/proxy?port=3003> . (Alternatively, in Cloud Shell, click on the Web Preview button and enter 3003 for the port.  If not using Cloud Shell, point a browser to `localhost:3000`.
+Open this URL to view the Grafana dashboard: <https://ssh.cloud.google.com/devshell/proxy?port=8080> . (Alternatively, in Cloud Shell, click the Web Preview button on the upper right corner and change the port to 8080 if necessary.  If not using Cloud Shell, point a browser to `localhost:8080`.
```

Comment:
I found the current cloud shell does not allow users to specify arbitrary port now. Instead, it provides a few common ports. I choose the port `8080` as the `3030` is not on the list.

![image](https://user-images.githubusercontent.com/91730/86322466-9fff7e80-bc6d-11ea-8bc6-dcd3614cfda1.png)